parcels/collection/collectionaos.py

```diff
@@ -692,7 +959,10 @@ def remove_multi_by_PyCollection_Particles(self, pycollectionp):
         each instance for its index (for random-access structures), which results in a considerable performance malus.
 
         For collections where removal-by-object incurs a performance malus, it is advisable to multi-remove particles
-        by indices or IDs.
+        by indices.
+
+        :arg pycollectionp: a Python-based collection (i.e. a tuple or list), containing Particle objects that are to
```

Comment:
what does the `np` in `pycollectionp` stand for? Is this a logical name for this argument?

Reply:
the `np` stand for `Numpy`, as each particle is represented by its numpy structured datatype (https://numpy.org/doc/stable/reference/arrays.dtypes.html; see `Particle` class). Thus, the argument here is a Python collection (e.g. list) of Particle instances stored in their respective Numpy format.

Reply:
OK, then add a `_` before `np`, so that the variable is called `pycollectio_np`?